src/constants/items.js

```diff
@@ -1,23 +1,26 @@
 export const ITEMS = {
-    WEAPONS: {
+    Weapon: {
         // Melee Weapons
-        DAGGER: {
+        Dagger: {
             name: "Dagger",
             cost: 50,
-            material: "SMITHINGMATS",
-            skill: "attack"
+            material: "ORE",
+            skill: "attack",
+            examineText: "Short but pointy"
         },
```

Comment:
In case you were wondering, I changed these names so that they match up with the class, I plan to make a Weapon class later so might as well adjust it now.

I could have Weapon be the same but the subcategories like DAGGER be in all caps, but I think it might look off. Thoughts?

Reply:
For the 2nd part we can see what we decide from our discussions. 

What do you mean by "Wouldn't we want them to be different than the class so we can easily tell the difference?", for this it would still be ITEMS.Weapon rather than just Weapon for the weapon class so I think that should be distinguishable enough. What would you suggest if we changed them?

Reply:
These are still constants so they should be capitalized. If we were pulling the attribute from a static weapon class then it should be Weapon.CONSTANT_ATTRIBUTE. By having it be ITEMS.Weapon then it makes the reader think we are pulling a class out of the ITEMS hat.

Reply:
Makes sense, we can remove this while moving it over into actual classes like we discussed though